packages/mermaid/src/diagrams/git/gitGraphRenderer.ts

```diff
@@ -812,9 +812,11 @@ const drawArrows = (
 
 const drawBranches = (
   svg: d3.Selection<d3.BaseType, unknown, HTMLElement, any>,
-  branches: { name: string }[]
+  branches: { name: string }[],
+  DEFAULT_GITGRAPH_CONFIG: any
```

Comment:
Please use variable names matching our convention. 
There will be lots of changed lines, but that's okay. 

```suggestion
  config: MermaidConfig['gitGraph']
```